### ISSUE TITLE
"InternalCompilerError: " Without info in `formattedMessage`

### ISSUE THREAD

**Issue by yann300** [ethereum/solidity#3563]
Hello anyone can help me. I write next code and I recive result "InternalCompilerError: "
```
pragma solidity ^0.4.19;

library KeyLib 
{
    uint constant NumberCombinationsCount4 = 11;
    uint constant NumberCombinationsCount5 = 26;
    uint constant NumberCombinationsCount6 = 57;

    struct NumberIndex
    {
        bytes4 indexes;
        uint level;
    }

    
    function fillIndexMap4(NumberIndex[] indexMap) 
        public
    {
    }
}
```
more info:
```
"{"component":"general","formattedMessage":"InternalCompilerError: \n","message":"Internal compiler error (/root/project/libsolidity/codegen/CompilerUtils.cpp:726):","severity":"error","type":"InternalCompilerError"}"
```

**Comment by yann300** [comment]
was https://github.com/ethereum/browser-solidity/issues/1091

**Comment by federicobond** [comment]
I cannot reproduce this on the latest version. It says:
```
test.sol:16:28: Error: This type is only supported in the new experimental ABI encoder. Use "pragma experimental ABIEncoderV2;" to enable the feature.
    function fillIndexMap4(NumberIndex[] indexMap)
                           ^--------------------^
```

**Comment by chriseth** [comment]
This has been fixed in 0.4.21.
